script/tool/lib/src/drive_examples_command.dart

```diff
@@ -180,30 +190,35 @@ Tried searching for the following:
 
   Future<bool> pluginSupportedOnCurrentPlatform(
       FileSystemEntity plugin, FileSystem fileSystem) async {
+    final bool isAndroid = argResults[kAndroid];
+    final bool isIOS = argResults[kIos];
     final bool isLinux = argResults[kLinux];
     final bool isMacos = argResults[kMacos];
+    final bool isWeb = argResults[kWeb];
     final bool isWindows = argResults[kWindows];
-    final bool isIOS = argResults[kIos];
-    final bool isAndroid = argResults[kAndroid];
+    if (isAndroid) {
+      return (isAndroidPlugin(plugin, fileSystem));
+    }
+    if (isIOS) {
+      return isIosPlugin(plugin, fileSystem);
+    }
     if (isLinux) {
       return isLinuxPlugin(plugin, fileSystem);
     }
     if (isMacos) {
       return isMacOsPlugin(plugin, fileSystem);
     }
+    if (isWeb) {
+      return isWebPlugin(plugin, fileSystem);
+    }
     if (isWindows) {
       return isWindowsPlugin(plugin, fileSystem);
     }
-    if (isIOS) {
-      return isIosPlugin(plugin, fileSystem);
-    }
-    if (isAndroid) {
-      return (isAndroidPlugin(plugin, fileSystem));
-    }
     // When we are here, no flags are specified. Only return true if the plugin
     // supports Android for legacy command support. TODO(cyanglaz): Make Android
     // flag also required like other platforms (breaking change).
     // https://github.com/flutter/flutter/issues/58285
     return isAndroidPlugin(plugin, fileSystem);
   }
 }
+//  - flutter drive -v --driver=test_driver/integration_test.dart --target=integration_test/example_test.dart
```

Comment:
Remove?

Reply:
Oops. Can you tell I needed an extra paste buffer? 😬